java_src/foam/nanos/audit/AuditOAO.java

```diff
@@ -36,26 +36,24 @@ private String formatMessage(FObject obj, Map values) {
   }
 
   @Override
-  public void setProperty(X x, String name, Object value) {
-    User       user     = (User) x.get("user");
-    NanoLogger logger   = (NanoLogger) x.get("logger");
-    FObject    obj      = getDelegate().get(x);
-    Object     objectId = ((PropertyInfo) obj.getClassInfo().getAxiomByName("id")).f(obj);
-    Map        values   = new HashMap();
-
+  public FObject setProperty(X x, String name, Object value) {
+    User user = (User) x.get("user");
+    NanoLogger logger = (NanoLogger) x.get("logger");
+    FObject obj = getDelegate().get(x);
+    Object objectId = ((PropertyInfo) obj.getClassInfo().getAxiomByName("id")).f(obj);
```

Comment:
Okay done. I went back and changed instances in my code where it was using the getAxiomByName method of getting & setting properties as well.